nameservice/tutorial/03-types.md

```diff
@@ -29,4 +29,9 @@ To start your SDK module, define your `nameservice.Whois` struct in the `./x/nam
 
 <<< @/nameservice/nameservice/x/nameservice/types/TypeWhois.go
 
+**Note:** after updating your `TypeWhois.go` file and reloading your app `starport serve` some compiling erros might appear due to change of some attribute types from `string` to `sdk.Coins`. Perform the following changes: 
+- `./x/nameservice/client/txWhois.go` - find and replace `string(argsPrice)` by `sdk.Coins(argsPrice)`
```

Comment:
Is this `./x/nameservice/client/txWhois.go` -  `/cli` or `/rest`?

Reply:
Good catch - I'm removing this as it is covered by further steps in the tutorial due to file changes :)